### DISCUSSION TITLE
Outputs

### DISCUSSION THREAD

**Discussion by jinca** [idaholab/moose#19303] [Q&A General] [answered]
Hello,
Hope everyone is doing well.
I am launching jobs in batch, so I am only directing the outputs to a file .out.
May I ask you how I can see the other output files?
I am talking about the files .e, and the file that shows the final calculations. (e.g. Temperature gotten on each element of the mesh).
Thanks in advance!
Julita

**Comment by GiudGiud** [comment, marked as answer]
Hello

What exactly is the concern?
The outputs are generated in the folder of the input file. 
If you are worried about them overwriting each other, you can pass a different file name using the file_base parameter in the Outputs block. 
So from the command line you can pass Outputs/file_base=fileXXX (no space) to each run numbered XXX so that the results don't overwrite each other.

Guillaume

**Comment by jinca** [comment]
Hi Guillaume, thanks for replying.
I know that the outputs are .out and they are placed in the same folder of the input file
I want to know where is the .e file to see the image of the simulation
Another question is where is the final result on each point of the mesh. I use log_view but only shows convergence message and thee number of iterations are different for each run. I put the same conditions and every time I run it, the iteration number changes. It is not reproducible. Thanks.

**Reply by GiudGiud** [reply]
The .e files (exodus) are also generated in that folder. If they are not for you, you can add exodus=true to the Outputs block

**Reply by jinca** [reply]
Hi Giudguid, that sounds a great advice, I will try it tomorrow and let you know to close this :) In case you can tell me if there is an option to see the final result of the simulation in a 'log file', it would be great. For example, if I am resizing the mesh, I want to see if the results are the same to be compared and confirm accuracy. Thanks again!

**Reply by GiudGiud** [reply]
oh you want to compare between mesh refinement?

If you can boil it down to a few numbers, then use Postprocessors and compare them at different levels of refinement. 
If you cannot and need to be comparing fields of values, then the solution is to load the two solutions in MOOSE or paraview, from two different exodus files, and compare them there. MOOSE has a bunch of projection options to compare the fine & coarse solutions or the fine OR the coarse mesh

**Reply by jinca** [reply]
Hello, 
I got this output:
*** Warning ***
/~/input.i:144: unused parameter 'Outputs/pgraph/exodus'

This is my block in my input file:
[Outputs]
  [pgraph]
     type = PerfGraphOutput
     exodus=true
     execute_on = 'initial final'
     level = 1
  []
[]

Thanks in advance again!

**Reply by GiudGiud** [reply]
```
[Outputs]
  exodus=true
  [pgraph]
  type = PerfGraphOutput
  execute_on = 'initial final'
  level = 1
  []
[]
```

is more correct

the perfgraph wont help you do comparisons. I think wrt comparisons, my previous answer should help

**Reply by jinca** [reply]
Hello dear Giud, what do you mean by wrt?

**Reply by GiudGiud** [reply]
with regards to
